### PR TITLE
fetch tutorial docs from master branch of electron/electron

### LINKED ISSUE #81: Surface improvements to tutorials on the website more quickly
## Problem

The website currently displays docs from the latest stable version of Electron. The stable version doesn't change very often, and sometimes there are weeks of beta versions before the next stable bump.

While it makes sense for the website's API documentation to strictly match the current stable version of Electron, the tutorial content found in `docs/tutorial` is generally not Electron-version-specific.

Currently we fetch all docs from `latest`:

Files | Source Commitish
--- | ---
`/docs/**/*.md` | electron@latest
`locale.yml` | electron.atom.io#master

Instead, I propose that we fetch `docs/tutorial/**` straight from Electron's master branch:

Files | Source Commitish
--- | ---
`/docs/api/**/*.md` | electron@latest
`/docs/development/**/*.md` | electron@latest
`/docs/tutorial/**/*.md` | electron#master
`locale.yml` | electron.atom.io#master

cc @leo, whose eagerness to see https://github.com/electron/electron/pull/10251 deployed on the website prompted this idea.

### PR DESCRIPTION
**Pull request by zeke**
Fixes #81 

This PR updates the data collection process a bit:

**API docs** still come from the `latest` (in npm dist-tag parlance) version of Electron, where latest does not mean most recent, but "stable". In other words, the version people will get when they `npm install electron`

**Tutorials** are any markdown files in `docs/` that are not in `docs/api`. They are now pulled from the master branch of electron/electron, and the commit SHA of master at the time of this collection is exported with the module so we can display it on the website. There's also a new `i18n.date` timestamp exported so we can display how recently the docs were updated on the site.

To Do:

- [x] Ignore (or include?) the new images directory. See a proposed workaround at https://github.com/electron/electron/pull/11314
- [x] Make sure version data is explicit, now that some docs are from #master and some are from a specific release / git tag.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -221,3 +231,13 @@ describe('i18n.electronLatestStableTag', () => {
...
   })
 })
+
+describe('i18n.date', () => {
+  it('exists', () => {
+    i18n.date.should.be.a('string')
+  })
+
+  it('is a date', () => {
+    i18n.date.should.match(/^20/)
```
Comment (vanessayuenn) [inline test/index.js]: Maybe we can do something like `new Date(i18n.date).toString().should.not.equal('Invalid Date')` to verify it's a date?

**Comment by zeke** [comment]
It was a chicken-egg issue. Build process needed some new package.json metadata, which I've added manually to kick things off. In the future, `electronLatestStableTag` and `electronMasterBranchCommit` will be updated automatically by the `collect` script.

**Comment by zeke** [comment]
@vanessayuenn this is ready for another look  👀

**Comment by vanessayuenn** [comment]
Sweet! This is awesome 🎉

**Pull request merged**